src/main/java/racingcar/view/RacingCarLog.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.view;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+
+public class RacingCarLog {
+
+    private RacingCarLog() {
+    }
+
+    public static StringBuilder printWinner(StringBuilder gameLog, Cars cars) {
+        List<String> winnerList = cars.getWinners().stream()
+            .map(Car::getName)
+            .collect(Collectors.toList());
+        return gameLog.append(String.join(",",winnerList));
```

Comment:
아래와 같이 변경할 수 있을 것 같아요! 😃
```suggestion
        String winners = cars.getWinners().stream()
            .map(Car::getName)
            .collect(Collectors.joining(","))
        return gameLog.append(winners);
```